<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemDialog</class>
 <widget class="QWidget" name="ItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>L1 Title</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Collection Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leCollectionName"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Item Type</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="cbL1Language"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Media URI</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="leL1Title">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>L2 Title</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>L2 Language</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>L2 Language</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="leL2Title">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QComboBox" name="cbL2Language"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Collection No</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="leMediaURI"/>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>RadioButton</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="leCollectionNo"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>247</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Content</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QsciScintilla" name="teL1Content">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QsciScintilla" name="teL2Content">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCopy">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSplit">
       <property name="text">
        <string>Split</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
